dali/test/python/test_utils.py

```diff
@@ -543,3 +543,19 @@ def get_files(path, ext):
       if re.match(f".*\.{ext}", f) is not None
   ]
   return audio_files
+
+
+def _test_omitted():
+    print("Omitting tests suite for Python3.8+ serialization")
```

Comment:
Please move this to the relevant test module or parameterize it. The name suggests a generic utility, but it produces a very specific message.

Reply:
done